<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateManager</class>
 <widget class="QDialog" name="TemplateManager">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag template manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tag templates</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="NewBtn">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="templateList"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="ExportBtn">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="EditBtn">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="DuplicateBtn">
     <property name="text">
      <string>Duplicate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="ImportBtn">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="DeleteBtn">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="FinishBtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>templateList</tabstop>
  <tabstop>DeleteBtn</tabstop>
  <tabstop>NewBtn</tabstop>
  <tabstop>DuplicateBtn</tabstop>
  <tabstop>EditBtn</tabstop>
  <tabstop>ExportBtn</tabstop>
  <tabstop>ImportBtn</tabstop>
  <tabstop>FinishBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
